python/semantic_kernel/connectors/memory/qdrant/qdrant_utils.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import datetime
+import uuid
+import struct
+
+import qdrant_client
+from qdrant_client.models import Filter, FieldCondition, MatchValue
+
+from numpy import array, linalg, ndarray, zeros
+
+from semantic_kernel.memory.memory_record import MemoryRecord
+
+from typing import Optional
+
+"""
+Utility function(s) for Qdrant vector database to support Qdrant Semantic Kernel memory implementation.
+"""
+
+
+def guid_comb_generator() -> str:
+    """
+    Generate a GUID-comb identifier.
+
+    Returns:
+        str: A GUID-comb identifier.
+    """
+
+    guid_bytearr = bytearray(str(uuid.uuid4()), "utf-8")
+
+    # Get the current time in 100-nanosecond intervals since 1/1/1
+    # Convert the time to a byte array
+    currenttime = datetime.datetime.utcnow()
+    daystickssec = (currenttime - datetime.datetime(1970, 1, 1)).total_seconds() * 1000
+    currtickssec = currenttime.microsecond / 1000
+
+    daysbytes = struct.pack(">i", int(daystickssec))
+    currbytes = struct.pack(">i", int(currtickssec))
+
+    daysbytearr = bytearray(daysbytes)
+    currbytearr = bytearray(currbytes)
+
+    daysbytearr.reverse()
+    currbytearr.reverse()
+
+    return str(guid_bytearr)
+
+
+def compute_similarity_scores(
```

Comment:
This method shouldn't be needed for qdrant since the calculations are done service-side

Reply:
Okay I'll remove it completely.

Reply:
Addressed and resolved in https://github.com/microsoft/semantic-kernel/pull/919/commits/2d79e60727ea2dc9dd267bb35b298c069c32b5f3